guide/english/certifications/javascript-algorithms-and-data-structures/debugging/use-the-javascript-console-to-check-the-value-of-a-variable/index.md

```diff
@@ -3,8 +3,23 @@ title: Use the JavaScript Console to Check the Value of a Variable
 ---
 ## Use the JavaScript Console to Check the Value of a Variable
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/certifications/javascript-algorithms-and-data-structures/debugging/use-the-javascript-console-to-check-the-value-of-a-variable/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+In order to open up the Javascript Console of your Browser (chrome,firefox,safari,opera,edge) press: [here](https://guide.freecodecamp.org/certifications/javascript-algorithms-and-data-structures/debugging/understanding-the-differences-between-the-freecodecamp-and-browser-console)
```

Comment:
As mentioned by @RandellDawson, there shouldnt be a reference to the challenge which the camper hasnt encountered yet (atleast for this one).

You can instead write couple of instructions?

Reply:
Hello @ezioda004  yeah sure I can write something extra no problem but if you want to specify regarding what would you like me to write one extra paragraph etc?

Reply:
@AlexandreLeFou Yes, just make it as concise as possible.  Looking forward to your addition!

Reply:
@AlexandreLeFou Any progress?